package.json

```diff
@@ -13,6 +13,8 @@
     "grunt-contrib-clean": "~0.5.0",
     "assemble": "~0.4.37",
     "grunt-contrib-copy": "~0.5.0",
-    "grunt-prettify": "~0.3.4"
+    "grunt-prettify": "~0.3.4",
```

Comment:
We should probably run an update on development dependencies since we're changing the packages and it's been a while since it was last updated.